### PR TITLE
Added declarations for 'get-stdin'.

### PR DESCRIPTION
**Pull request by DanielRosenwasser**
Added declarations for [get-stdin](https://www.npmjs.com/package/get-stdin).

### CODE REVIEW

File: get-stdin/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": ["node"],
```
Comment (DanielRosenwasser) [inline get-stdin/tsconfig.json]: Is this okay, or do I need an explicit `/// <reference` to Node/ @andy-ms @RyanCavanaugh

Comment (andy-ms) [inline get-stdin/tsconfig.json]: `/// <reference types="node" />` is better, since we don't publish `tsconfig.json`.

**Comment by andy-ms** [comment]
Please rebase onto the latest `types-2.0` to fix tests. Fixed by #13133.

**Comment by andy-ms** [comment]
There are a few extra commits here. Could you try rebasing again?

**Pull request merged**